fix: missing chevron in conversation container in shared folder

mgarbin opened the pull request:
refs: IRIS-4415

mgarbin committed:
fix: missing chevron in conversation container in shared folder

refs: IRIS-4415

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/zextras/carbonio-mails-ui?pullRequest=409) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/zextras/carbonio-mails-ui?pullRequest=409) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/zextras/carbonio-mails-ui?pullRequest=409) it.</sub>

giuliano176 reviewed (commented):
since this bug has got graphical implications, could you add a small test for it?

gnekoz commented:
The test should check that the chevron icon is visible if there is more than 1 message on the conversation, and also that there is no chevron if the conversation has just 1 message.
A similar test can be found at https://github.com/zextras/carbonio-mails-ui/blob/38bec115c1870e58944359e190a86541805c2c0d/src/views/app/folder-panel/conversations/tests/conversation-list-item-component.test.tsx#L27

mgarbin commented:
@giuliano176 I don't know how to write a test, probably i need a month to understand how to do that ( reading as example test example under test folder... )

mgarbin commented:
@gnekoz yep, but probably we need to rewrite this test to support bot shared and local folder...

mgarbin committed:
Merge branch 'devel' into IRIS-4415-fix-share-conversation-chavron

mgarbin committed:
Merge branch 'devel' into IRIS-4415-fix-share-conversation-chavron

gnekoz commented:
> @gnekoz yep, but probably we need to rewrite this test to support both shared and local folder...

In this test case the parent folder is not relevant. The conversation and its messages are generated inside the test (with no regard for the given folder) and do not have the same issue (discrepancy between number of messages and the _n_ field).

gnekoz commented:
> @giuliano176 I don't know how to write a test, probably i need a month to understand how to do that ( reading as example test under test folder... )

I linked the existing test which can be used as model for the new test, with just  a couple of changes. I can support you for the creation of the new test

mgarbin committed:
test: add test for cheveron

- verify if it exist on multi message conrsation
- verify if it doesn't exist on single message conversation

mgarbin committed:
test: fix code format on chevron test case

gnekoz committed:
Merge branch 'devel' into IRIS-4415-fix-share-conversation-chavron
